frontend/components/text2text/results/Text2TextList.vue

```diff
@@ -228,6 +255,28 @@ export default {
     window.addEventListener("keyup", this.keyUp);
   },
   methods: {
+    initializeViewSetting() {
+      this.itemNumber = 0;
+      this.editionMode = false;
+      if (this.annotationEnabled) {
+        if (this.annotations.length) {
+          this.sentencesOrigin = "Annotation";
+        } else if (this.predictions.length) {
+          this.sentencesOrigin = "Prediction";
+        }
+      } else {
+        if (this.predictions.length) {
+          this.sentencesOrigin = "Prediction";
+        } else if (this.annotations.length) {
+          this.sentencesOrigin = "Annotation";
+        }
+      }
```

Comment:
This could be the default recordsViewSettings value in `models/Text2TextRecord` class